scss/config/_global-settings.scss

```diff
@@ -12,9 +12,102 @@ $decanter-default-settings: (
   breakpoint-helper-selector: #breakpoint-helper,
 );
 
-/// @name Bourbon Settings
-///
-/// @group settings
-$bourbon: (
-  "modular-scale-ratio": 1.333,
+$neat-grid: (
```

Comment:
This seems to be a duplicate of what is included in stanford_basic. Why has it been copied here without any changes? The config.scss roll up file is including stanford_basic's file. Also, if you have not already, please update your copy of stanford_basic as the global variables for $grid-columns and $gutter have been removed in favor of a real time retrieve function. See: https://github.com/SU-SWS/stanford_basic/blob/8.x-1.x/scss/config/_global-settings.scss#L81